update-dependencies/Program.cs

```diff
@@ -108,7 +108,7 @@ private static IDependencyUpdater CreateDependencyUpdater(string path)
             {
                 Path = path,
                 BuildInfoName = "Cli",
-                Regex = new Regex($@"ENV DOTNET_SDK_VERSION [\d\.]*-(?<version>{s_config.CliReleaseMoniker}-[^\r\n]*)"),
+                Regex = new Regex($@"ENV DOTNET_SDK_VERSION [\d\.]*-(?<version>{s_config.CliReleaseMoniker}-[\d]*)\r\n"),
```

Comment:
Consider using the '?' quantifier for '\r', in case the file just has LF instead of CRLF:  `\r?\n`